src/Kopernicus/RuntimeUtility/RuntimeUtility.cs

```diff
@@ -276,6 +276,13 @@ void Start()
                         Destroy(city.gameObject);
                     }
                 }
+                var stockLaunchSitesField = PSystemSetup.Instance.GetType().GetField("stocklaunchsites", BindingFlags.NonPublic | BindingFlags.Instance);
+                LaunchSite[] stockLaunchSites = (LaunchSite[])stockLaunchSitesField.GetValue(PSystemSetup.Instance);
+                var updateStockLaunchSites = stockLaunchSites.Where(site => !Templates.RemoveLaunchSites.Contains(site.name)).ToArray() ;
```

Comment:
Tbh. I would prefer using the full type name instead of `var`, but thats no a real issue

Reply:
Very different styles, I actually deeply regret the type declaration on line 280. However, this project's style is to explicitly declare the type so I will fix it on lines 279 and 281 :)